react/Checkbox/styles.styl

```diff
@@ -1,4 +1,28 @@
 @require '../../stylus/components/forms'
+@require '../../stylus/settings/palette'
 
 .c-input-checkbox
     @extend $checkbox
+
+// Cancel the default behaviour for input checkbox
+.c-input-checkbox--svg span::after
```

Comment:
Why did we add the class if we cancel the effect? Is it because we want the background but take control over the tick / dash ?

Reply:
It's for backwards compatibility. 
I don't touch the "old way" so it's still working and add a class for the "new way" to cancel the old way.
When we decide to definitively remove the old way, we can remove this class and this chunk of css along with the css part of the old way.

I couldn't find a smoother way.